subsys/settings/Kconfig

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+choice SETTINGS_BACKEND
+	default SETTINGS_ZMS_LEGACY if (SETTINGS_ZMS_DOWN && ZMS)
```

Comment:
I second Kamil (in a different comment) that `ZMS_UP` and `ZMS_DOWN` don't bring any value. If you want `SETTINGS_ZMS` to be selected by default if `ZMS` is enabled, you can just remove this line and `SETTINGS_ZMS_LEGACY` will be used only when selected explicitly. Or am I missing something?